.packit.yaml

```diff
@@ -7,10 +7,17 @@ upstream_tag_template: v{version}
 
 srpm_build_deps:
   - make
+  - sudo
+  - rpm-build
 
 actions:
   fix-spec-file:
-    - bash .packit.sh
+    - bash -c './.packit.sh'
```

Comment:
but packit.sh is empty, why is it?

Reply:
you are right, I am testing to see if something was broken.